src/Emulator/Peripherals/Peripherals/Miscellaneous/STM32_PWR.cs

```diff
@@ -0,0 +1,146 @@
+using System.Collections.Generic;
+using Antmicro.Renode.Core;
+using Antmicro.Renode.Core.Structure.Registers;
+using Antmicro.Renode.Peripherals.Bus;
+using Antmicro.Renode.Logging;
+using Antmicro.Renode.Utilities;
+
+namespace Antmicro.Renode.Peripherals.Miscellaneous
+{
+	[AllowedTranslations(AllowedTranslation.ByteToDoubleWord | AllowedTranslation.WordToDoubleWord)]
+	public sealed class STM32_PWR : IDoubleWordPeripheral, IKnownSize
+	{
+		public STM32_PWR(Machine machine)
+		{
+			var registerDictionary = new Dictionary<long, DoubleWordRegister>
+			{
+				{(long)Registers.PowerControl, new DoubleWordRegister(this, 0xFCC00)
+					.WithFlag(0, name: "LPDS")
+					.WithFlag(1, name: "PDDS")
+					.WithFlag(2, FieldMode.Read | FieldMode.WriteOneToClear, name: "CWUF", valueProviderCallback: _ => { return false; }, writeCallback: (_, value) => {
+							if (value)
+							{
+								wufValue.Value = false;
```

Comment:
Can `wufValue.Value` be ever `true` in this model?

Reply:
You're right. Can I change it to a TaggedFlag then? Does it still have the same behaviour?

Reply:
Changed to TaggedFlag and the behaviour is the same.